Cleanup and simplification

simonverret opened the pull request:
This should make the code easier to use for newcomers to the project.
- Stripped down everything but the Data generator files and neural net training notebook
- Major refactoring of the data generator file (easier to naviguate)
- Removed everything related to hyperparameter search
- Removed everything related to multiple temperature experiment
- Removed Datasets file (although retains compatibility)
- Removed everything related to Lorentz Comb data
- Removed unnecessary dependencies
- Replaced `argparse` command line interface with google `Fire` (one-line CLI)
- Added progress tracking of data generation
- local RandomState instead of using numpy's global RandomState

simonverret committed:
scratch most code (simplified data generator)

simonverret committed:
Completed the new dataset workflow
- updated readme
- choose beter name conventions
- added support for dataset parameters file
- adapted the notebook

simonverret committed:
changed the rescale convention

simonverret committed:
notebook on statistics

simonverret committed:
slight improvements

simonverret committed:
use RandomState instead of global random

simonverret committed:
Save s only if rescaling

amstremblay commented:
Merci beaucoup Simon,
André-Marie

De : Simon Verret ***@***.***>
Envoyé : 22 avril 2022 10:48
À : simonverret/deep_continuation ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [simonverret/deep_continuation] Cleanup and simplification (PR #12)


This should make the code easier to use for newcomers to the project.

  *   Stripped down everything but the Data generator files and neural net training notebook
  *   Major refactoring of the data generator file (easier to naviguate)
  *   Removed everything related to hyperparameter search
  *   Removed everything related to multiple temperature experiment
  *   Removed Datasets file (although retains compatibility)
  *   Removed everything related to Lorentz Comb data
  *   Removed unnecessary dependencies
  *   Replaced argparse command line interface with google Fire (one-line CLI)
  *   Added progress tracking of data generation
  *   local RandomState instead of using numpy's global RandomState

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/simonverret/deep_continuation/pull/12<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622063157%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Z1otDXjrJIki7BuHZRdK%2BQskrGPyqAClD%2F4LJxWBrxc%3D&reserved=0>

Commit Summary

  *   c096e50<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Fcommits%2Fc096e50b2d6809044d4411c9cf84c11630519d59&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622063157%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kfzs5DFTOZBkAwyzqTAkwfmWMY%2BlKuClznrayBYyVXU%3D&reserved=0> scratch most code (simplified data generator)
  *   c717428<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Fcommits%2Fc717428cc939e90315d1cf09d98463691c599c32&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622063157%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=AU9kmJR9KmfL0ltQG38%2FztqxjEboQJ%2BOhlzK8jtwQBc%3D&reserved=0> Completed the new dataset workflow
  *   67b73f0<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Fcommits%2F67b73f00674863bb05dbffc6a4946400ad74439b&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622063157%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=jgRHFwKrytaRKfHzxNlKYnu6HNSbZnc4X27bOKnta0k%3D&reserved=0> changed the rescale convention
  *   7314942<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Fcommits%2F7314942f5fea869186aec527dcd5dec3e4530d2e&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622063157%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wLKNm2kVDjTyCJWzdOh%2FYZ%2F1UzdDNRAX0bJGPZl%2Buhs%3D&reserved=0> notebook on statistics
  *   ea09701<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Fcommits%2Fea09701e3d69ce7953798d74981c2583c9d4069b&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622063157%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Trv5PQ%2F8QnWhaaZGQotyGJmWl7AVHzC%2BFPBUttt4Zwc%3D&reserved=0> slight improvements
  *   6a9ce0c<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Fcommits%2F6a9ce0c39026e4de1740a44ba230baedf8bb7d9e&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=axlwWY4uSVdELgzbTnUWwqcZE7sADXAJIDuwIKiWfmk%3D&reserved=0> use RandomState instead of global random

File Changes

(58 files<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=WMvTkBY1jFMipooh48lFt2Pgy8qr85lJ%2BtJK%2BjZMNfw%3D&reserved=0>)

  *   M .gitignore<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6L5lqEY5tqSNp11OfOall1Yp4w%2BlXF9%2BP8z4uUT6D44%3D&reserved=0> (94)
  *   M README.md<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3w1ScbcnOugISsM8yONBZUpcLvvC4npga7Vuwvd0kjY%3D&reserved=0> (135)
  *   D bin/make_submit_files.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-84e5aca980d37dd6b1386492b2fdd2987febf51e82c709683977668e39ef5e19&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GMO2IveqiMhLEYE5WCT1T%2Bk4jzU71ARZ%2B7kAh5fO0%2Fk%3D&reserved=0> (109)
  *   D bin/submit_genB.sh<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-e238abc9b1024934968fee9da2889bf5fcf8b5da206c577c86c358f9efb7d040&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=O9MffUQXFw9JqA1aIZtTdJzjXSZbwtvl9fWHxyKrxOw%3D&reserved=0> (32)
  *   D bin/submit_genF.sh<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-a6a0c5e579d43fbbf3008e9f9abc938d5ee7f1ef0fbc8b47c78dfd1fffb2e0ac&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=APu7Faa%2BICoj9iuDyYnQV46lhf1AzrUkLAcnAuodzcw%3D&reserved=0> (32)
  *   D bin/submit_genG.sh<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-ddb58f5a30b647286eeaea4dda1b102ad802e4810157dbf5640b957ccc2e4c70&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=a4zOWYIGIfCvbYHQcvAKYFijXj9slru5lQ5ofvssefI%3D&reserved=0> (32)
  *   A deep_continuation/conductivity.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-3de381e5bf03fc06eff2153fce7bb0c72fe965eb42ae31ea21074a8ac391e5d4&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BvY4Sj32JGEKHEQTdzem4fs4Jm9%2FFxA%2FRw6yU%2B4UbT0%3D&reserved=0> (104)
  *   D deep_continuation/data.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-5e7d2901740a32fc3a6d922c907246bc09636408c42c48ddd369ea4af2b8d1ee&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=SdCXnq2sIHbg2FA8FKNVdE56BPXidUNs4QLXMiofPwQ%3D&reserved=0> (113)
  *   D deep_continuation/data/B1_train.json<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-dcffe69708e45fd5c450f424d62a7f78515bded899e8aa9566136cbd2d33c038&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=hVmuNvkSsUcrOQeSdXAxohIXTFrfH%2FnIrSuJ6x0b6Xc%3D&reserved=0> (14)
  *   D deep_continuation/data/Fournier_train.json<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-fa78b46a3d5f4e5ca27ff805e3616019741e94721646d20cefe06e864170c68b&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=IjdvgWhWOu%2BWovIiCv7euea8S0VZaRj5szxXEgQHrdc%3D&reserved=0> (13)
  *   D deep_continuation/data/Fournier_valid.json<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-2442cce4efe06fef32926e8d47fbd4911c3a8de4b91a80f47e3be148e2262308&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=SZkMySn%2B0tYy1dvOuVBLi7MZZfPW%2BMVf%2BP2Bmsml14c%3D&reserved=0> (13)
  *   D deep_continuation/data/G1_train.json<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-0a89ecaad6d61927756701abd7087026af2cca54026eb38bd6cdbb2cade351a4&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=es3jou2e6cR1%2FPC5YJdBrEE2C7x69lu7tUCEXrdrqKE%3D&reserved=0> (12)
  *   D deep_continuation/data/G1_valid.json<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-2f918c21cfed7a8f1f77dda284a5275733adaf02d92736d4a4ff77f7940fa62d&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6AvSQ%2B8vtX2858Qkt2DqyCTZG3uqkeZmlebCtx8cet4%3D&reserved=0> (12)
  *   R deep_continuation/data/default.json<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-bcbdc9689233d46397cfa68104942ae39dc1ee2769a8e709b0416e054e32031e&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=H6q0sZTLCUnVzoa05v39ejR%2BXUsrVQIfFR7A%2BV%2Bu1Mc%3D&reserved=0> (5)
  *   D deep_continuation/data_generator.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-a95c707202dd38679f7641e75a82c9b52f87f6faf5834f7ee94057ca36c18583&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=npygp6wEcDTX3bDCWTgprpCaWbhzN2z1WABCVkiM9sc%3D&reserved=0> (245)
  *   A deep_continuation/dataset.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-e6f3a13852e21a08feefe5c30bc2e9df026b9463c4a7040ec398a46bfee9dde9&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=rV1ECel1VMxccZ%2F2cQSgkVPAYzB5v24EUCqls8Pp0KU%3D&reserved=0> (121)
  *   A deep_continuation/dataset_statistics.ipynb<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-e944ed6a24645e83d8e72345d49a46bc7f563143c79294aa3d18f216e00280fe&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lpSdTBFryeCMdz4XXcxYaQPcB6D6rKFX9GOudXV5C2o%3D&reserved=0> (183)
  *   A deep_continuation/distributions.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-7c8627c2aafd30040c5a0a3446345718046962d5fc7e63ae0cb7e7faa21c939e&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=pdDu9g6yScqxa0j4Rrb7KUIMN86hQ6IuzsZivb4B9fs%3D&reserved=0> (333)
  *   D deep_continuation/function_generator.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-cd88405d59d4e8ebaa238044f52a0d4d56e897fa4721cd2301b9f77f27a78d4b&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=UAB9577PlS2ESQVYvoNmUmNKrvRgjtLtcd7xIdTbUn0%3D&reserved=0> (590)
  *   D deep_continuation/infinite_data.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-24e6be11dc0478b264df5467acb62ad3d05a5d417f447652bdaddad69bde19a5&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=iI1%2Fk65sYSwkb4jtTYGo1ZOi2GgRVGm%2FRYTYCgRlFIA%3D&reserved=0> (117)
  *   D deep_continuation/monotonous_functions.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-a017e8d5b48ebc81a880dae9f5af04c494ddb7751807ce56ec740adf67baeb48&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2B85MxNpNxxuVZtaaGiREvh%2BXmFBjZFds7BxauGK1%2BO8%3D&reserved=0> (467)
  *   D deep_continuation/plots/OpticalConductivity05.dat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-5678bb92602a59ea1b94cf69224ad73d2b8b21b631568bbb94155e7cdf0b6d40&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=jWSnhkuNNEs90CKZ05TaBmUs7I620og0vQqe3m2pok4%3D&reserved=0> (278)
  *   D deep_continuation/plots/OpticalConductivity1.dat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-0825931275033122da2023b50bab28f62e603fc4de2533a8b7f15139e86779a8&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=T668G8VTUzRDuU1qgv5rhONlwoOs%2BqhzeHHRz%2BnPnwY%3D&reserved=0> (278)
  *   D deep_continuation/plots/OpticalPolarization.dat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-5d024b8c3c0d3c731b4c8ec61769aae70609e80b47c95d0923ac310818f3f0ef&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Qt7jzcj90JjnAsUbXp2A9a5SGOlqMynDCsWq9S8cWn8%3D&reserved=0> (148)
  *   D deep_continuation/plots/Pi_figure_corrected.csv<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-5c070f317709c26566badfadae7e1962ebdca7b42d70709131da84cffebff351&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2TDxmqgqDkMSe8SH8ZU1KebuQBsjk%2BzX0psc1%2F9CA1g%3D&reserved=0> (1)
  *   D deep_continuation/plots/Pol.txt<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-cbdfa1659f2c7c00f4ab2d5593546e629b2806900ee407c908bd722da0fe3a34&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622219373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=9GDUe4lFE6%2BFUEoB3X6OSrwUvYJQ4TgyPIPcoQ1Hnos%3D&reserved=0> (128)
  *   D deep_continuation/plots/all_best.csv<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-2aa327dd64e708bfdec711bd2bdee8ab45ef68dadb0fea8d36b6f47beee66caa&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=7ch0jXUHUBPYg7f49JEo9ACFjykA65tOY2hHqgiFDeo%3D&reserved=0> (476)
  *   D deep_continuation/plots/figure_data.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-08d436b810608a10d3666e7a91f37b81655e78ac46e9a95496fb2b360fc4d827&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=I10NZ1vqW8oWQSVRJnnU4XBg7OSTXrKuF6O7LGqeuPo%3D&reserved=0> (70)
  *   D deep_continuation/plots/figure_fit.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-65239fbdf7162581c2fb9f0dee9473c5d7f9140bbffd1fc8fa2739d7808e7b5e&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=AjH%2FvxiqJ%2FntCIc3%2BPLjeQQ3uXnql%2BRFVuFU1hBWv%2FM%3D&reserved=0> (170)
  *   D deep_continuation/plots/figure_optimal_spectral_function_tem0.05_alpha1.59e+07.dat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-14d39fbcb83b752d0f9764f90bd57b985772c527f3416009ebe69124875e5434&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GWvpdXEYe1q8MD3uSYLSQP7wGiRrzWr7ytSkOpE%2BGk0%3D&reserved=0> (114)
  *   D deep_continuation/plots/ill_simple.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-bfe94267f2bacc4d692d03207ed38821edfefa7d26408676a2b512ebc9c9dd9b&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=X9bDsUV%2Fv12MisgBrRfjNz%2FqqRTtP6DUP1a%2FilErxLs%3D&reserved=0> (400)
  *   D deep_continuation/plots/optimal_spectral_function_tem0.03333_alpha1.59e+07.dat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-1712102c70692033d068f492a81fb9a3cd1cc84859fc8e721c956dbbc07d00d0&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=8%2BS1cj%2FpitG8xJvqhMwqqgGZkWMppLlD6L76LDcS9hU%3D&reserved=0> (114)
  *   D deep_continuation/plots/optimal_spectral_function_tem0.05_alpha1.73e+06.dat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-9ca8e46429c0d7bbf378c82fd1d0be9e220cb509a77a0a13847638d542ab36fb&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xnogLCoTBNhGyahTLsVJmzkFhbbgLi%2BlQlRoz1q5AH4%3D&reserved=0> (114)
  *   D deep_continuation/plots/plot_hyper.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-29249551e4ac97897b33aba6fb6431a953dd0fb5d6dc6e8a5d85d3e47e2ffffc&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=j%2FDy%2F04mCQtryzx1GYWz5aJCuo05y9vDgTKfBCFwyn4%3D&reserved=0> (78)
  *   D deep_continuation/plots/plot_results.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-7a859cd8348bdec007f2565d4d8888a8c452f3511d597b8e3491a4a9683e7011&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=5r%2FcdA7WjFqICrdcgMI5rzN1dFYok6%2FvtgN2bnmYBeE%3D&reserved=0> (79)
  *   D deep_continuation/plots/plot_training.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-88f40e0cb099b3745b58f6436790b09b050afb90ddc018360850ae652f7c9397&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wQ0cIk1PtyK%2FwzuBgOo8ms01EGJ5Bpsy%2FV4NjuZMQ10%3D&reserved=0> (37)
  *   D deep_continuation/plots/position_effect.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-fb5923b3b426335747bb8e7e2887e2f3a288dcfa26d83f5022e5a7fb1b8bd1c7&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=daLgpRmOLGTQZs9P13a4GJPoWu8DeMbzKtD5iOmNyQo%3D&reserved=0> (51)
  *   D deep_continuation/plots/requirements.txt<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-e07aa22b155aedeee5a12e6675931d574639dd2d7cd1f1e2c5aa5baf4504a8a4&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ucRhwflRQ5QHdEnQjVopVV4elkyXF45tcz9NJxc2vMM%3D&reserved=0> (36)
  *   D deep_continuation/plots/reza_SigmaRe_scaled_4.000000.csv<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-7d758391292ad711da98cbb89af1e45e7b527afa93dacb23a0097014f5c4c853&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wCTPjRpO34Nt3tT3ZI4FLcoFmnO1vID%2FznJTUtcxPZI%3D&reserved=0> (20)
  *   D deep_continuation/plots/reza_pol.txt<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-0dcb5cc58956eccfad26e33fdc629f5b8b1f4efc8bc4268ef3d34dd7ca5b4c50&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ddromYNbfIItv05exQFUA3bSi2EmP1xScFHCUPXsl50%3D&reserved=0> (128)
  *   D deep_continuation/plots/scale.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-c76e538b8efd6dbf1f4028ba433695cce56f76e570f9fc0563962bb2e7a0f1c9&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=3DMtSnVhtbg8%2F9vk6E%2BxKS1nPlMoJ%2Fim7Xolgmng5GY%3D&reserved=0> (214)
  *   D deep_continuation/plots/sigma_figure.csv<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-e52508b08c2938032cbc02d029b78d0556f5db0dfbc33a0ab421c8927e9d086a&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=UCFJYguAQH1gVB%2BEv7%2FRMj0uyRLxbHO0aCZzGoHxnYs%3D&reserved=0> (512)
  *   D deep_continuation/plots/test_plots.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-5d7c7a3575c174f7ad33f53d91b4df53bab42ce2c9c5188c6fe9c7c3ed492934&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=mIbCFMUlF7aNHVIcRSha3AcFeeSI8mgHqsuB29nsNrU%3D&reserved=0> (148)
  *   D deep_continuation/plots/wandb_data.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-b5dd9c749aa80a1fc3e6280b3513517dfcc7bdb7d8fffca7f85f2eed5ba8d2fb&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FdyeprhrDDw8qqigGLuipYTCD%2F%2BBWh1BRB7a4ViS9ec%3D&reserved=0> (426)
  *   A deep_continuation/plotting.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-64f7490a39a9ae026583636084ed0c3575b9ee2e3de7fb32efd7bb664178f3eb&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=1AJimFFhV7X7nUzday09DDNoQUwtMGcxqH37LtehEbU%3D&reserved=0> (110)
  *   D deep_continuation/random_search.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-690c09bcf82d353b27bc0acfcae109018ffeccfc4ae419072c698b61d9acfbdb&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ZSmKLQNB1uh7UXoXVyvsDiZbEAYEuDqpctBKQbjYYGk%3D&reserved=0> (216)
  *   D deep_continuation/train.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-6f1441f3063b9bb886e359c8a7e63e05f60414fab9a5bc36512a6e7237c84a09&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=B4yd1upcnPTQbeDy1c3Qxk3aagzCJz%2FgtKOAMkG6otE%3D&reserved=0> (454)
  *   M deep_continuation/tutorial_retrain_best.ipynb<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-dc76a90957bff3a1f693092e24ff90a0cfed2b09a73a652ffe099dc1a13e8084&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=B1WKRVbR6TLTJbMchSK3eZlTotPCpV9sLodHE0U7AI4%3D&reserved=0> (192)
  *   D deep_continuation/utils.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-1a5dd6f89a6654e9262ae2283a2d6fccfd25c0b91a94c24be174a9d65319a1b3&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=S8c%2Bj4zl9bTrv2RQMfi8NsNrxVB1LbL78p5GLjRV5WU%3D&reserved=0> (62)
  *   D deep_continuation/wandb_utils.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-e6c6d8f022fbbfc353664d31e00b5952fe96c08e56f930251da446ca5e2589e5&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=5X%2BLP6sEsw96EMxKVx0yPHW%2BMEADIdKg%2FtlMg11t6SI%3D&reserved=0> (128)
  *   D docs/Makefile<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-cdec797dddf8fc387304501d6bd4f318e1e9067c6fae012dfa8b69fd85fd3248&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2yF7j87OSu%2FfFIA5Qm%2F4yJWO64PZQNk1QfX3fUQKcJ0%3D&reserved=0> (20)
  *   D docs/make.bat<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-eced23cf23b339d5a7dcd2246cb8fb515b1fa640975b585950c75ae19ba55feb&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OW8roCOUFvrbbI0F3ZfT3eeVVWJF8gQvoOZigLOWgOw%3D&reserved=0> (35)
  *   D docs/source/conf.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-008dcb3426febd767787b1521f1fe33086313b927ea37eaab86df5fa88a51698&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=1UHtHsytagHnH5cwmiu7Ti5bnT77A3JqGSiJmDSEqmw%3D&reserved=0> (56)
  *   D docs/source/deep_continuation.rst<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-19d53e74028e944f9740c379528a901dbe902d34c67e6f6e3774dd41bbced715&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ZmguZlq0xfZjgXy8Ovo7Llcef4PpBOg0S6nf8lZMkEA%3D&reserved=0> (85)
  *   D docs/source/index.rst<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-4eec6cec5f6fab1548b85433ea8ca81315ae165db4b7f84019f287df9015699f&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622376158%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2BXeOw9v9ktOWZNVdzz5k9gi6U9w5GHabHIpn07gAqgk%3D&reserved=0> (20)
  *   D docs/source/modules.rst<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-bc1ef7e9cb9f11ba7705ec3a60d320a08679d8e9da78e97613db7b8f8618432c&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GJ0I3Zo3bFiiBdsqxjPlP8hV%2FtxZbzBvsbzzQcqrs%2BI%3D&reserved=0> (8)
  *   D docs/source/setup.rst<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-3af4a7b2c92d8c2a3e6686483cc960cc28428c3bd79c805346e33b68e801b1ec&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FxYqYICshKZo7suIAZgZQSjxPqnHGMu6u%2FCCnoWQ0Pc%3D&reserved=0> (7)
  *   M setup.py<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12%2Ffiles%23diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=SrLroT8ajHg2tZbFXg92VW3PnonsJbYUYmWNINBEvOI%3D&reserved=0> (7)

Patch Links:

  *   https://github.com/simonverret/deep_continuation/pull/12.patch<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12.patch&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=cAdMXpeeoG0CoP4kxzrrcZ8xmeJQmjug4Zt84n%2BNaFg%3D&reserved=0>
  *   https://github.com/simonverret/deep_continuation/pull/12.diff<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12.diff&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Og2gl5MLDo2Y9eRg4W%2B4d%2F03GyfBTLU4yvhADM7G2fs%3D&reserved=0>

-
Reply to this email directly, view it on GitHub<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsimonverret%2Fdeep_continuation%2Fpull%2F12&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=staGiWIh1mIWsJXMbamb%2BPzU2k73eyQE2kD9KovFVPc%3D&reserved=0>, or unsubscribe<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAE2GYL5GEAN74EUV7WESJRLVGK3YXANCNFSM5UCRM64Q&data=05%7C01%7Ctremblay%40physique.usherbrooke.ca%7C91411ddf1eab490c505108da246f0d0c%7C3a5a8744593545f99423b32c3a5de082%7C0%7C0%7C637862356622531818%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0ox8IbQW%2FipPuLvVDZ4wE%2BhmCXv44Mq47IajJaRShZ8%3D&reserved=0>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.******@***.***>>

simonverret committed:
added report

simonverret committed:
finished new version of the report

simonverret committed:
removed commented code

simonverret committed:
minimalist tests

simonverret committed:
renaming, and updates to docs

simonverret committed:
Added tests for the usecases of the readme file
- fixed an issue regarding the skipped filenames (rescaling changes Pi)
- fixed an issue where no plot would show if plot was true without one of basic, scale, or infer being also true.

simonverret committed:
More restructuration
- dropped pathlib in favor of os.path
- made more robust to windows path
- use `.npy` instead of `.txt` for faster datafile
- rerun all tests and  notebooks (with changes due to the rescaled vs spurious naming mistake)

simonverret committed:
attempt at showing spurious correlations

simonverret committed:
use scale file instead of local fit to illustrate spurious correlation

simonverret committed:
split get_sigma and get_pi

simonverret committed:
moved the logic of data generation in dataset.py

simonverret committed:
skip Pi computation when possible
- added tests for skipping files
- added a readme to the tests with command to recreate the files
- corrected a bug in the last version

simonverret committed:
slight clarifications to the logic of generation

simonverret committed:
cosmetic adjustments

simonverret committed:
multi temperature working

simonverret committed:
Added multi-temperature data preparation

simonverret committed:
improvements to the report text

simonverret committed:
removed the inferring s and T section

simonverret committed:
finished training details and usage details

simonverret committed:
updated report and saved a static pdf version for sharing

simonverret committed:
multi_std and multi_beta + new filenames
- TODO: training notebook
- TODO: graph for multi std
- TODO: 2D histograms

simonverret committed:
Add save fixstd feature

simonverret committed:
fix tests

simonverret committed:
Random seed for beta and fixstd

simonverret committed:
Removed infer axis from plotting

simonverret committed:
improved statistics notebook

simonverret committed:
histograms in the dataset statistics notebook

simonverret committed:
histograms in the dataset statistics notebook

simonverret committed:
Merge branch 'task_simplification' of https://github.com/simonverret/deep_continuation into task_simplification

simonverret committed:
Adding a "name" arg, to bypass path if desired

simonverret committed:
updated training notebook for new filenames
